bundles/binding/org.openhab.binding.myq/src/main/java/org/openhab/binding/myq/internal/GarageDoorData.java

```diff
@@ -44,57 +41,47 @@
 	 * @param logData
 	 *            Boolean to determine if devicedata should be logged.
 	 */
-	public GarageDoorData(String deviceStatusData, boolean logData) {
+	public GarageDoorData(String deviceStatusData) {
 		ObjectMapper mapper = new ObjectMapper();
 		try {
 			JsonNode rootNode = mapper.readTree(deviceStatusData);
 			int ReturnCode = rootNode.get("ReturnCode").asInt();
-			logger.debug("myq ReturnCode: " + Integer.toString(ReturnCode));
+			logger.debug("myq ReturnCode: {}", Integer.toString(ReturnCode));
 
 			if (ReturnCode == 0) {
-				this.success = true;
 				if (rootNode.has("Devices")) {
 					JsonNode node = rootNode.get("Devices");
 					if (node.isArray()) {
-						if (logData) {
-							logger.info("Chamberlain MyQ Devices:");
```

Comment:
Typically we use TRACE logging for stuff like this.  It can be changed on the fly without restart the runtime (logback.xml)